iree/compiler/Dialect/VMLA/IR/VMLAOps.td

```diff
@@ -178,6 +178,12 @@ def VMLA_TransposeOp : VMLA_ElementTypeOp<"transpose", [VMLA_IncludeShapes]> {
     VMLA_Shape:$dst_shape,
     VMLA_AnyTypeAttr:$element_type
   );
+
+  let assemblyFormat = [{
```

Comment:
I'm a little worried that `<>` will be confusing because it's usually used to specify types and here we're actually wrapping other SSA values. Not sure though.

Reply:
yeah, <> is used for attribute contents. () seems fine.